composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "illuminate/support": "5.5.* || 5.6.* || 5.7.* || 5.8.*"
+        "illuminate/support": ">5.7"
```

Comment:
This may be better as `~5.8.0 || ^6.0` as using an unbound version constraint is [less secure](https://getcomposer.org/doc/faqs/why-are-unbound-version-constraints-a-bad-idea.md)?

Reply:
Good note! Updated!